CQUI.modinfo

```diff
@@ -299,7 +233,6 @@
         <File>Assets/UI/Panels/unitpanel.xml</File>
         <File>Assets/UI/PartialScreens/citystates.lua</File>
         <File>Assets/UI/PartialScreens/citystates.xml</File>
-        <File>Assets/UI/Popups/boostunlockedpopup.lua</File>
```

Comment:
same here, I added this in 90bc06981fe68f8c9724e107ec5d7971786d866f

Reply:
changes made